[2023-10-07 12:30:04,198] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: sales_mart.update_d_customer manual__2023-10-07T12:28:45.671748+00:00 [queued]>
[2023-10-07 12:30:04,205] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: sales_mart.update_d_customer manual__2023-10-07T12:28:45.671748+00:00 [queued]>
[2023-10-07 12:30:04,206] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-10-07 12:30:04,206] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-10-07 12:30:04,207] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-10-07 12:30:04,216] {taskinstance.py:1377} INFO - Executing <Task(PostgresOperator): update_d_customer> on 2023-10-07 12:28:45.671748+00:00
[2023-10-07 12:30:04,218] {standard_task_runner.py:52} INFO - Started process 2926 to run task
[2023-10-07 12:30:04,220] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'sales_mart', 'update_d_customer', 'manual__2023-10-07T12:28:45.671748+00:00', '--job-id', '88', '--raw', '--subdir', 'DAGS_FOLDER/sprint3.py', '--cfg-path', '/tmp/tmpmrfshc5b', '--error-file', '/tmp/tmp2hv8o55k']
[2023-10-07 12:30:04,221] {standard_task_runner.py:80} INFO - Job 88: Subtask update_d_customer
[2023-10-07 12:30:04,258] {task_command.py:369} INFO - Running <TaskInstance: sales_mart.update_d_customer manual__2023-10-07T12:28:45.671748+00:00 [running]> on host d964b2b568e5
[2023-10-07 12:30:04,303] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=student@example.com
AIRFLOW_CTX_DAG_OWNER=student
AIRFLOW_CTX_DAG_ID=sales_mart
AIRFLOW_CTX_TASK_ID=update_d_customer
AIRFLOW_CTX_EXECUTION_DATE=2023-10-07T12:28:45.671748+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-10-07T12:28:45.671748+00:00
[2023-10-07 12:30:04,311] {base.py:68} INFO - Using connection ID 'postgresql_de' for task execution.
[2023-10-07 12:30:04,315] {sql.py:315} INFO - Running statement: insert into mart.d_customer (customer_id, first_name, last_name, city_id)
select customer_id, first_name, last_name, max(city_id) from staging.user_order_log
where customer_id not in (select customer_id from mart.d_customer)
group by customer_id, first_name, last_name, parameters: None
[2023-10-07 12:30:04,317] {sql.py:324} INFO - Rows affected: 0
[2023-10-07 12:30:04,327] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=sales_mart, task_id=update_d_customer, execution_date=20231007T122845, start_date=20231007T123004, end_date=20231007T123004
[2023-10-07 12:30:04,353] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-10-07 12:30:04,381] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
